foursight_core/react/api/react_routes.py

```diff
@@ -17,15 +17,15 @@ def __init__(self):
     @route("/auth0_config", authorize=False)
     def reactapi_route_auth0_config() -> Response:  # noqa: implicit @staticmethod via @route
         """
+        Returns Auth0 configuration for authentication (from Portal).
         Note that this in an UNPROTECTED route.
-        No-env version of above /{env}/auth0_config route.
         """
         return app.core.reactapi_auth0_config(app.current_request.to_dict())
 
     @route("/cognito_config", authorize=False)
     def reactapi_route_cognito_config() -> Response:  # noqa: implicit @staticmethod via @route
         """
-        Returns AWS Cognito configuration (from environment variables and/or Secrets Manager).
+        Returns AWS Cognito configuration for authentication (from environment variables or Secrets Manager).
```

Comment:
Where would one go to figure out the details of how env variables vs secrets manager are triaged?

Reply:
In cognito.get_cognito_oauth_config. Added comment to this effect here.